tests/unit/featurizers/test_molgraph.py

```diff
@@ -3,11 +3,11 @@
 from rdkit import Chem
 
 from chemprop.featurizers import (
-    MoleculeMolGraphFeaturizer,
     SimpleMoleculeMolGraphFeaturizer,
     MolGraph,
     MultiHotAtomFeaturizer,
 )
+from chemprop.featurizers.molgraph.base import MolGraphFeaturizer
```

Comment:
Probably fine to import this explicitly, but then why not also do `from chemprop.featurizers.molgraph.molecule import SimpleMoleculeMolGraphFeaturizer` explicitly. I think it would be better to group `MolGraphFeaturizer` with the other imports.